.travis.yml

```diff
@@ -32,6 +32,7 @@ env:
     - COMPOSER_FLAGS=""
     # By default test with Symfony 2.8, some test are done against 3.4 for forward compat testing.
     - SYMFONY_VERSION="~2.8.52"
+    - PHP_IMAGE=ezsystems/php:7.2-v1
```

Comment:
I'd go for 7.1

Maybe even 7.0 for 6.13 as minimum version is 5.6 _(but that will slow it down to much so let's not go there)_

Reply:
I'd be ok with this change (though I still prefer 7.2) - let's see what others think 😉

Reply:
5.6 _might_ have the risk of timing out. But I agree we should have coverage of it, maybe we can have it covered on the nightly jobs as opposed to on PR's?

Either way, we should sync up the PHP version used by test client on Travis env, and the PHP version used by app inside container. And we should spread the functional test runs across versions.

e.g:
```
    - php: 7.1
      env: BEHAT_OPTS="--profile=rest --tags=~@broken --suite=fullJson" RUN_INSTALL=1 SYMFONY_ENV=behat PHP_IMAGE=ezsystems/php:7.1-v1
```